app/controllers/application_controller.rb

```diff
@@ -1,5 +1,13 @@
+require 'twitter_client'
+
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  before_filter :set_twitter_client
```

Comment:
Do we need this to happen _everywhere_, or is this something that we could set in specific controllers? (But leave it _defined_ here)